.circleci/diagnostics.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+INPUT=
+OUTPUT=
+NODES=''
+BROKEN_NODES=''
+
+function usage {
+  echo "Usage:"
+  echo "diagnostics.sh -n ./nodes.csv -o /tmp/outputfile"
+  echo "the csv should be of the form <operator>,<if disabled "true" or empty>,<[dns|ip][:port|]>"
+  echo "if no port is provided, port 80 is assumed."
+}
+
+while getopts "n:o:h" arg; do
+  case $arg in
+    n)
+      INPUT=$OPTARG
+      ;;
+    o)
+      OUTPUT=$OPTARG
+      ;;
+    h)
+      usage;
+      exit 0;
+      ;;
+  esac
+done
+
+[ ! -f $INPUT ] && { echo "$INPUT file not found"; usage; exit 99; }
+[ "$OUTPUT" != "" ] && touch $OUTPUT || { echo "$OUTPUT file not writable"; usage; exit 99; }
+
+while read operator broken ip
```

Comment:
I think we can check the connection to the fullnode AC and filter out those not connected from the test below.

Reply:
that's exactly what this script does with the clustertests that are running with --diag

Reply:
this script also lets the csv mark a node as broken so that we don't have constant failure notifications (if we don't want them).